Reminder-App/Reminder-App/ReminderDetailEditDataSource.swift

```diff
@@ -76,17 +85,31 @@ class ReminderDetailEditDataSource: NSObject {
         switch section {
         case .title:
             if let titleCell = cell as? EditTitleCell {
-                titleCell.configure(title: reminder.title)
+                titleCell.configure(title: reminder.title) {
+                    title in
+                    self.reminder.title = title
+                    self.reminderChangeAction?(self.reminder)
+                }
             }
         case .dueDate:
             if indexPath.row == 0 {
-                cell.textLabel?.text = reminder.dueDate.description
+                cell.textLabel?.text = formatter.string(from: reminder.dueDate)
             } else if let dueDateCell = cell as? EditDateCell {
-                    dueDateCell.configure(date: reminder.dueDate)
+                dueDateCell.configure(date: reminder.dueDate) {
+                    date in
```

Comment:
for closures, move the argument (date) and **in** keyword after opening braces like
`dueDateCell.configure(date: reminder.dueDate) { date in

//CODE

}
`